lnrpc/lightning.proto

```diff
@@ -1202,6 +1205,12 @@ message SendManyRequest {
 
     // The strategy to use for selecting coins during sending many requests.
     CoinSelectionStrategy coin_selection_strategy = 9;
+
+    /*
+    A manual fee rate set in sat/kweight that should be used when crafting the
+    transaction.
+    */
+    uint64 sat_per_kw = 10;
```

Comment:
received a good idea from @starius to visually group all the feerate values in the protobuf file (even though the indices will not be monotonically increasing) I think it is a better way from a design perspective.